docs/_articles/en/user-guide/metadata-operations.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Metadata Operations
+lang: en
+---
+## Overview
+
+
+Say you want to change an existing Apex class or add a custom object to your production org. You first make and test these changes in a scratch org, check for conflicts, and then deploy the changes to another org for your team to access them. This ability to move changes between different Salesforce development environments such as scratch orgs and sandboxes, for various stages of testing before launching into production, is key to Salesforce development. Use these metadata operations to deploy your local changes to an org as a part of your ALM process. With conflict detection turned on, you don't have to worry about accidentally overwriting others' changes in the org. You can track changes by easily viewing changes between your local files and your default org at any time. If you’re working in a team, you can retrieve the latest changes in your org on to your local Salesforce project. 
```

Comment:
```suggestion
This ability to move changes between different Salesforce development environments such as scratch orgs and sandboxes, for various stages of testing before launching into production, is key to Salesforce development. 
```
minor nit - this sentence has a ton of great info that's important to highlight, but it's a bit long so I'm getting a little lost. Could we rephrase or break it up a bit?

Reply:
oops not suggesting we delete the other sentences lol